assemblies/features/base/src/main/filtered-resources/resources/bin/client

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
```

Comment:
I would prefer:
1. directly refer `/bin/bash`
2. remove bash dependency.
It's what I tried at the beginning but due to some limitation and improvements, we introduced bash. I gonna check what's bash specific and fix that.